cmd/kubeadm/app/cmd/phases/certs.go

```diff
@@ -66,6 +66,25 @@ var (
 		` + cmdutil.AlphaDisclaimer)
 )
 
+// certsData defines the behavior that a runtime data struct passed to the certs phase should
+// have. Please note that we are using an interface in order to make this phase reusable in different workflows
+// (and thus with different runtime data struct, all of them requested to be compliant to this interface)
+type certsData interface {
+	Cfg() *kubeadmapi.InitConfiguration
+	CertificateDir() string
+	CertificateWriteDir() string
+}
+
+// NewCertsPhase returns the phase for the certs
+func NewCertsPhase() workflow.Phase {
+	return workflow.Phase{
+		Name:   "certs",
+		Short:  "Certificate generation",
+		Long:   "Certificate generation for the whole cluster",
```

Comment:
we should just remove the Long description here:
```
Short: "Certificate generation for the whole cluster"
```

long should be used if it's really long - multiple lines.

Reply:
Updated